src/components/messenger/chat/index.test.tsx

```diff
@@ -181,15 +181,20 @@ describe(DirectMessageChat, () => {
   describe('mapState', () => {
     describe('canLeaveRoom', () => {
       it('is false when only when one other member', () => {
-        const state = new StoreBuilder().withActiveConversation(stubConversation({ otherMembers: [stubUser()] }));
+        const state = new StoreBuilder().withActiveConversation(
+          stubConversation({ otherMembers: [stubUser() as User] })
+        );
 
         expect(DirectMessageChat.mapState(state.build())).toEqual(expect.objectContaining({ canLeaveRoom: false }));
       });
 
       it('is false when user is admin', () => {
         const state = new StoreBuilder()
           .withActiveConversation(
-            stubConversation({ otherMembers: [stubUser(), stubUser()], adminMatrixIds: ['current-user-matrix-id'] })
+            stubConversation({
+              otherMembers: [stubUser() as User, stubUser() as User],
```

Comment:
hm, should I just add the missing props to `stubUser` and remove the `Partial` ?

Reply:
lol, good question. yes.